dds_registration/models.py

```diff
@@ -85,6 +85,24 @@ def __init__(self, *args, **kwargs):
         self._original_username = self.username
 
 
+class Membership(models.Model):
+    user = models.ForeignKey(User, related_name='registrations', on_delete=models.CASCADE)
+    started = models.IntegerField(default=lambda: date.today().year)
+    until = models.IntegerField(default=lambda: date.today().year)
```

Comment:
While trying to create migrations, I'm getting errors for these lambda's:

`Cannot serialize function: lambda`.

Reply:
OK, let's switch to:

```python
def this_year():
    return date.today().year
```

And then `default=this_year`.